src/components/ebay-progress-bar-expressive/progress-bar-expressive.stories.ts

```diff
@@ -0,0 +1,184 @@
+            table: {
+                defaultValue: {
+                    summary: "[]",
+                },
+            },
+        },
+        size: {
+            type: "enum",
+            control: {
+                type: "select",
+            },
+            options: ["large", "medium"],
+            description: "Message text size",
+            table: {
+                defaultValue: {
+                    summary: "large",
+                },
+            },
+        },
+        isLoading: {
+            control: {
+                type: "boolean",
+            },
+            description: "Whether the component should be visible",
+            table: {
+                defaultValue: {
+                    summary: "true",
+                },
+            },
+        },
+    },
+};
+
+export const Default = Template.bind({});
+Default.args = {};
+Default.parameters = {
+    docs: {
+        source: {
+            code: tagToString("ebay-progress-bar-expressive", Default.args),
+        },
+    },
+};
+Default.parameters = {
+    docs: {
+        source: {
+            code: tagToString("ebay-progress-bar-expressive", Default.args),
+        },
+    },
+};
+
+export const WithSingleMessage = Template.bind({});
+WithSingleMessage.args = {
+    messages: [{ text: "We're processing your order" }],
+};
+
+export const WithMessages = Template.bind({});
+WithMessages.args = {
+    messages: exampleMessages,
+};
+WithMessages.parameters = {
+    docs: {
+        source: {
+            code: tagToString(
+                "ebay-progress-bar-expressive",
+                WithMessages.args,
+            ),
+        },
+    },
+};
+
+export const WithLongMessage = Template.bind({});
+WithLongMessage.args = {
+    messages: [
+        { text: "Messages should be one line..." },
+        { text: "Sometimes that's hard to guarantee, though.", duration: 2500 },
+        { text: "That's okay!" },
+    ],
+};
+WithLongMessage.parameters = {
+    docs: {
+        source: {
+            code: tagToString(
+                "ebay-progress-bar-expressive",
+                WithLongMessage.args,
+            ),
+        },
+    },
+};
+
+export const WithCustomTiming = Template.bind({});
+WithCustomTiming.args = {
+    messages: [
+        { text: "Display for 2 seconds", duration: 2000 },
+        { text: "Display for 3 seconds", duration: 3000 },
+        { text: "Display for 4 seconds", duration: 4000 },
+    ],
+};
+WithCustomTiming.parameters = {
+    docs: {
+        source: {
+            code: tagToString(
+                "ebay-progress-bar-expressive",
+                WithCustomTiming.args,
+            ),
+        },
+    },
+};
+
+export const Localized = Template.bind({});
+Localized.args = {
+    a11yText: "Cargando...",
+    messages: [
+        { text: "Espera..." },
+        { text: "Estamos procesando tu pedido", duration: 2000 },
+        { text: "Sólo un momento más" },
+    ],
+};
```

Comment:
One thing we have been trying to do is have a `localized.marko` template in examples, and importing that directly. 
I know we have this sprinkled everywhere, but going forward we want to do this method since its much more explicit when showing the code

See
https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-menu-button/menu-button.stories.ts#L237-L241 and https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-menu-button/examples/icon-with-text.marko

Reply:
Ah yes, that is more explicit! I'll update, though it looks like I will also need to explicitly pass args so the Storybook control panel has the correct initial values. Are there plans to update the `buildExtensionTemplate` utility to automatically pull initial args from the example component?